lib/features/mailbox_dashboard/domain/state/remove_composer_cache_state.dart

```diff
@@ -1,15 +1,15 @@
 import 'package:core/presentation/state/failure.dart';
 import 'package:core/presentation/state/success.dart';
 
-class RemoveComposerCacheSuccess extends UIState {
+class RemoveLocalEmailDraftSuccess extends UIState {
```

Comment:
we dont need to have `id` to see what local draft was removed?

Reply:
It is updated right on the UI so we don't need it.